src/h5web/dataset-visualizer/DimensionMapper.tsx

```diff
@@ -26,7 +27,7 @@ function DimensionMapper(props: Props): JSX.Element {
 
     return (
       // eslint-disable-next-line react/no-array-index-key
-      <Fragment key={dimension}>
+      <div key={key || dimension} className={styles.sliderWrapper}>
```

Comment:
Is `key` really needed?

Reply:
I have a warning in the console telling it is needed if I don't put it.

Reply:
Sorry I meant the local variable `key`, not the attribute.

Reply:
Removed !